app/server/lib/HostedMetadataManager.ts

```diff
@@ -1,6 +1,8 @@
-import {DocumentMetadata, HomeDBManager} from 'app/gen-server/lib/homedb/HomeDBManager';
+import {DocumentMetadata} from 'app/gen-server/lib/homedb/HomeDBManager';
 import log from 'app/server/lib/log';
 
+export type SetDocsMetadataFunc = (metadata: { [p: string]: DocumentMetadata }) => Promise<any>;
```

Comment:
Can you add a comment on what this type is for.

Reply:
Added a comment, and slightly renamed the type for a bit more clarity.